### PR TITLE
Feature/add ui for statusbay helm chart

### PR DESCRIPTION
**Pull request by cregev**
I have created an additional part for the UI in StatusBay Helm chart.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,6 +45,9 @@ Parameter | Description | Default
 | **Ingress** |
 `ingress.api.annotations` | The annotations used in ingress | `{}`
 `ingress.api.host` | The host of Statusbay ingress api | `api.statusbay.domain`
+`ingress.ui.annotations` | The annotations used in ingress for the UI | `{}`
+`ingress.ui.host` | The host of StatusBay ingress UI | `statusbay.domain`
+
 | **Service** |
 `service.api.type` | The type of api service to create  | `NodePort`
 `service.api.annotations` | The annotations used in api service | `{}`
```
Comment (kaplanelad) [inline README.md:53]: Add UI service doc

File: README.md [hunk 2]
```diff
@@ -54,6 +57,16 @@ Parameter | Description | Default
 | **Service Account** |
 `serviceAccount.create` | Specifies whether a ServiceAccount should be created | `true`
 `serviceAccount.name` | The name of the ServiceAccount to use  | ``
+| **UI** |
+`ui.create` | If true, UI server will be deploy | `true`
+`ui.replicas` | The replica count of the UI webserver | `2`
+`ui.annotations` | The annotations to be used in the UI deployment | `{}`
+`ui.image.repository` | UI container image repository | `similarweb/statusbay`
```
Comment (kaplanelad) [inline README.md]: Should be `similarweb/statusbay-ui`

File: templates/ingress/ingress-ui.yaml [hunk 3]
```diff
@@ -0,0 +1,21 @@
+{{- if .Values.ui.create -}}
```
Comment (kaplanelad) [inline templates/ingress/ingress-ui.yaml]: Change filename to UI, remember the API also

File: templates/ui/deployment.yaml [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+      serviceAccountName: {{ template "statusbay.serviceAccountName" . }}
+      containers:
+        - name: {{ .Chart.Name }}-ui
+          image: "{{ .Values.ui.image.repository }}:{{ .Values.ui.image.tag }}"
+          imagePullPolicy: {{ .Values.ui.image.pullPolicy }}
+          resources:
+            {{- toYaml .Values.ui.resources | nindent 12 }}
+          env:
+            - name: API_URL
+              value: "http://{{ template "statusbay.fullname" . }}-api/api/v1/kubernetes"
```
Comment (kaplanelad) [inline templates/ui/deployment.yaml]: Remove the suffix

File: templates/ui/deployment.yaml [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+            - name: GELF_ADDRESS
+              value: "{{ toYaml  .Values.ui.application.log.gelf_address | nindent 12 }}"
```
Comment (kaplanelad) [inline templates/ui/deployment.yaml]: Fix the one value

File: templates/ui/deployment.yaml (hunk 4 above)
Comment (kaplanelad) [reply templates/ui/deployment.yaml]: and remove the HTTP prefix

File: templates/ui/deployment.yaml [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+            - name: LOG_LEVEL
+              value: "{{ toYaml  .Values.ui.application.log.level | nindent 12 }}"
```
Comment (kaplanelad) [inline templates/ui/deployment.yaml]: Yaml to one value

File: README.md [hunk 7]
```diff
@@ -54,6 +57,16 @@ Parameter | Description | Default
...
+`ui.image.tag` | UI container image tag | `dev`
+`ui.image.pullPolicy` | container image pull policy | `IfNotPresent`
```
Comment (kaplanelad) [inline README.md]: IfNotPresent -> always

File: templates/ingress/ingress-ui.yaml (hunk 3 above)
Comment (cregev) [reply templates/ingress/ingress-ui.yaml]: Done

File: README.md (hunk 7 above)
Comment (cregev) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (cregev) [reply README.md]: Done

File: README.md (hunk 1 above)
Comment (cregev) [reply README.md:53]: Done

**Review by kaplanelad — APPROVED** [review]

**Pull request merged**